nomad/structs/csi.go

```diff
@@ -273,24 +273,25 @@ func (v *CSIVolume) RemoteID() string {
 
 func (v *CSIVolume) Stub() *CSIVolListStub {
 	stub := CSIVolListStub{
-		ID:                 v.ID,
-		Namespace:          v.Namespace,
-		Name:               v.Name,
-		ExternalID:         v.ExternalID,
-		Topologies:         v.Topologies,
-		AccessMode:         v.AccessMode,
-		AttachmentMode:     v.AttachmentMode,
-		MountOptions:       v.MountOptions,
-		CurrentReaders:     len(v.ReadAllocs),
-		CurrentWriters:     len(v.WriteAllocs),
-		Schedulable:        v.Schedulable,
-		PluginID:           v.PluginID,
-		Provider:           v.Provider,
-		ControllersHealthy: v.ControllersHealthy,
-		NodesHealthy:       v.NodesHealthy,
-		NodesExpected:      v.NodesExpected,
-		CreateIndex:        v.CreateIndex,
-		ModifyIndex:        v.ModifyIndex,
+		ID:                  v.ID,
+		Namespace:           v.Namespace,
+		Name:                v.Name,
+		ExternalID:          v.ExternalID,
+		Topologies:          v.Topologies,
+		AccessMode:          v.AccessMode,
+		AttachmentMode:      v.AttachmentMode,
+		MountOptions:        v.MountOptions,
+		CurrentReaders:      len(v.ReadAllocs),
+		CurrentWriters:      len(v.WriteAllocs),
+		Schedulable:         v.Schedulable,
+		PluginID:            v.PluginID,
+		Provider:            v.Provider,
+		ControllersHealthy:  v.ControllersHealthy,
+		ControllersExpected: v.ControllersExpected,
```

Comment:
^ this line is the only addition, the rest of the diff is gofmt doing it's thing.